kernel/classes/datatypes/ezuser/ezuser.php

```diff
@@ -1804,6 +1816,19 @@ static function createHash( $user, $password, $site, $type, $hash = false )
         {
             $str = $password;
         }
+        else if ( $type == self::PASSWORD_BCRYPT )
+        {
+            if ( $hash ) {
+                if ( password_verify( $password, $hash ) ) {
+                    return $hash;
+                } else {
+                    return false;
+                }
+            } else {
```

Comment:
nitpick: CS. i know there are changes about this in ez5, but with legacy we are still applying old coding standards, right? If so, else should be in new line as you do in https://github.com/ezsystems/ezpublish-legacy/pull/1195/files#diff-9aabf5ca51952bbd469991c839db4f3eR1819